autoPVS1_from_VEP_vcf.py

```diff
@@ -1,48 +1,110 @@
 #!/usr/bin/env python
+"""
+Modified autoPVS1 wrapper script, as original did not work but overall scoring logic preserved.
+Overall goal is to classify "Null Variants".
+A good definition of what a PVS1 Null variant is: https://www.baylorgenetics.com/variant-classification/
+"""
 # -*- coding:utf-8 -*-
 # author: Jiguang Peng
 # created: 2019/6/27 19:13
 # Modified: 2022/10/05
-# Mod_auther: Miguel Brown
+# Mod_author: Miguel Brown
 
-import os
-import re
 import sys
-import random
-import string
 import argparse
-from collections import namedtuple
 import pysam
-from read_data_mod import trans_gene, gene_trans, gene_alias
 from read_data_mod import transcripts_hg38
 from pvs1 import PVS1
 from utils import get_transcript, vep_consequence_trans, VCFRecord
-__version__ = 'v0.2.0'
+__version__ = 'v2.0.0'
 
 
-lof_type = ['frameshift', 'nonsense', 'splice-5', 'splice-3', 'init-loss']
-vep_lof_list = ['frameshift', 'stop_gained', 'splice_donor', 'splice_acceptor', 'start_lost']
-VAR = namedtuple('VAR', ('varid', 'gene', 'trans', 'canonical', 'pick', 'record'))
-
-
-class AutoPVS1:
-    """Run AutoPVS1"""
-    @staticmethod
-    def get_vcfrecord(vcf):
-        if all(hasattr(vcf, attr) for attr in ['chrom', 'pos', 'ref', 'alt']):
-            return vcf
-        elif re.match(r'^((?:chr)?[\dXYMT]{1,2})-(\d+)-([ATCG]+)-([ATCG]+)$', vcf, re.I):
-            v = vcf.split("-")
-            v[0] = re.sub('chr', '', v[0], flags=re.I)
-            return VCFRecord(v[0], int(v[1]), v[2], v[3])
-        else:
-            raise TypeError("Wrong VCF Record")
-
-
-    def run_pvs1(self):
-        pvs1 = PVS1(self.vcfrecord, self.consequence, self.hgvs_c, 
-                    self.hgvs_p, self.transcript, self.genome_version)
-        return pvs1
+def get_candidates(record_csq, csq_fields):
+    """
+    1. See if the tx candidate is canonical
+    2. If canonical, check to see if it exists in the tool transcript reference
+    3. Record the PICK value for rule 5 from pick_transcript
+    """
+    canon_idx = csq_fields.index("CANONICAL")
+    p_idx = csq_fields.index("PICK")
+    
+    pick_value = "1"
+    canon_value = "YES"
+    r_idx = csq_fields.index('RefSeq')
+    picked_csqs = []
+    tx_candidates = []
+
+    for csq in record_csq:
+        info_list = csq.split("|")
+        refseq_id = info_list[r_idx]
+        if info_list[p_idx] == pick_value:
+            pick = (get_transcript(refseq_id, transcripts_hg38), info_list)
+        if info_list[canon_idx] == canon_value:
+            if refseq_id != "":
```

Comment:
Cut down your indent
```suggestion
        if info_list[canon_idx] == canon_value && refseq_id != "":
```